src/Build.UnitTests/Construction/SolutionProjectGenerator_Tests.cs

```diff
@@ -155,7 +155,7 @@ public void SolutionFilterFiltersProjects()
                     @"
                 {
                   ""solution"": {
-                    ""path"": """ + solutionFile.Path.Replace("\\", "\\\\") + @""",
+                    ""path"": """ + Path.Combine(".", "SimpleProject", "SimpleProject.sln").Replace("\\", "\\\\") + @""",
```

Comment:
I don't understand this at all. What does this path.combine do?

Reply:
The test previously had an absolute path to the solution file. This is making it a relative path. Since the directory is also different from that with the solution filter file, this is essentially testing that MSBuild looks at the directory of the solution filter file in calculating the path to the solution file rather than using the current directory.

Reply:
ah. Well, it should definitely have a relative path. Can you _also_ add an assert that the current working directory is not the slnf directory?